cmake/scripts/get_latest_tag.sh

```diff
@@ -4,6 +4,11 @@
 # current HEAD
 #
 
+if ! git --version | grep -q "git version 2"; then
+  # This script requires version 2 or newer
+  exit 1
```

Comment:
i don't mind either way, but we may fallback to `git describe --tags --abbrev=0`